gsi/gssapi/source/library/init_sec_context.c

```diff
@@ -367,15 +372,65 @@ GSS_CALLCONV gss_init_sec_context(
 
         /*
          * TLS v1.3 sends 2 session tokens after the handshake is completed.
+         * If we are using a token source that sends one SSL token at a time
+         * (e.g. the file based token source in globus-gss-assist)
+         * these sessions tokens will not be seen by BIO_do_handshake before
```

Comment:
Small typo, but no need to rebase your PR:

"these sessions tokens" => "these session tokens"

Reply:
Thank you, I fixed the typo since I had to rebase one of the PRs anyway.